src/parallel-rustc.md

```diff
@@ -10,15 +10,27 @@ These next few sections describe where and how parallelism is currently used,
 and the current status of making parallel compilation the default in `rustc`.
 
 The underlying thread-safe data-structures used in the parallel compiler 
-can be found in `rustc_data_structures/sync.rs`. Some of these data structures
-use the `parking_lot` API.
+can be found in the `rustc_data_structures::sync` crate. Some of these data structures
+use the `parking_lot` module as well.
```

Comment:
I think you changed the wrong one ;)

`parking_lot` is a crate, `rustc_data_structures::sync` is a module.

Reply:
🤦‍♂️